### PR TITLE
MIA - FTWD - RAFAEL GUASP - Spotify Clone

### PR DESCRIPTION
**Pull request by rguasp**
not done need to tweak it around.Will submit new request on weeknd.

### CODE REVIEW

File: starter-code/index.html [hunk 1]
```diff
@@ -3,40 +3,81 @@
...
+</div>
+
+<div class="Hdmusic">
+<img src="./images/high-quality-icon.png" alt="Hdmusic">
+<h3>HD Music</h3>
+<p>Listen to music as if you</p>
+<p>were listening live</p>
+</div>
+
+<div class="StreamEverywere">
```
Comment (ianizaguirre) [inline starter-code/index.html]: Throughout this entire HTML page, you decided to name SOME of your class names using camel case? Please only use camel case in Javascript. The convention everyone follows for naming classes in HTML is to use all lower case and separate words with a hyphen.  So for example, your `"StreamEverywere"` should have been `"stream-everywere"` ... A few reasons why everyone follows this rule, is because CSS syntax is mostly case-insensitive. So to avoid errors others can have working on a team project with your code or reading your code, its best to just keep everything lower case and to separate words with a hyphen. It's best that you just realize that the best approach is the most compatible: use the same case in all code for a given CSS class, and don't create two or more CSS class names that differ only in case. It is also a good practice for readability. Also, even though there are other accepted ways to name classes in HTML, like using an underscore instead of a hyphen, for now lets stick with a hyphen.

Comment (ianizaguirre) [reply starter-code/index.html]: Also I think you misspelled this part of your class name:  "everywere"

File: starter-code/index.html [hunk 2]
```diff
@@ -3,40 +3,81 @@
...
+</div>
+</div>
+
+<div class="container2">
+  <div class="container2head">
+<h2>What’s on Spotify?</h2>
+<hr>
+</div>
+</div>
+<div class="container3">
```
Comment (ianizaguirre) [inline starter-code/index.html]: Please start getting into the habit of properly spacing/tabbing ALL of your divs, because all of your divs are on the same left side as the body tag, so now it is hard to follow where starting and closing tags start and end.

File: starter-code/index.html [hunk 3]
```diff
@@ -3,40 +3,81 @@
...
+</nav>
+<div class="container">
+  <div class="containertitle">
+<h1>Music for everyone.</h1>
+<p>Spotify is now free on mobile, tablet and computer.</p>
+<p>Listen to the right music, wherever you are.</p>
+</div>
+</div>
+
+<div class="container2">
```
Comment (ianizaguirre) [inline starter-code/index.html]: Calling things `"container1"` , `"container2"`  , `"container3"`  etc.  should be avoided. Names like `"container1"`  are way to general. Choose a better name since when we make div classes the plan is to reuse them eventually in our code in other places...So if the name is descriptive, it wont look out of place in another part of our HTML code when we reuse it, and a good name can tell us information.
For example:
 `  <div class="container"> `
could have been better written as something like:
 `  <div class="hero-wrapper"> `
or
 `  <div class="hero-container"> `

File: starter-code/css/main.css [hunk 4]
```diff
@@ -6,3 +6,131 @@ Green: #00B172
 White: #FFF
 
 */
+
+body{
+  margin: 0 auto;
```
Comment (ianizaguirre) [inline starter-code/css/main.css]: It would be a good habit to include a width in our body, so add: `width: 100%;`

File: starter-code/css/main.css [hunk 5]
```diff
@@ -6,3 +6,131 @@ Green: #00B172
...
+
+}
+.top-right li{
+  
+  display: inline-block;
+  margin: 0 10px;
+  
+  
+}
+nav{
```
Comment (ianizaguirre) [inline starter-code/css/main.css]: Properly space/tab your CSS rules so the properties are nested under the rule.

File: starter-code/index.html [hunk 6]
```diff
@@ -3,40 +3,81 @@
...
+<p>smartphone, tablet or</p>
+<p>computer</p>
+</div>
+</div>
+
+<div class="kanye-west">
+  <div class="kanye-west-wrapp">
+    <h2>It’s as yeezy as Kanye West.</h2>
+
+    <div class="search-browse-discover">
```
Comment (ianizaguirre) [inline starter-code/index.html]: "search-browse-discover" is a little too long of a class name, try to make it smaller

File: starter-code/index.html [hunk 7]
```diff
@@ -3,40 +3,81 @@
...
+<img src="./images/devices-icon.png" alt="StreamEverywere">
+<h3>Stream Everywhere</h3>
+<p>Stream music on your</p>
+<p>smartphone, tablet or</p>
+<p>computer</p>
+</div>
+</div>
+
+<div class="kanye-west">
+  <div class="kanye-west-wrapp">
```
Comment (ianizaguirre) [inline starter-code/index.html]: `<div class="kanye-west-wrapp">`, 
I think you either mean to write `<div class="kanye-west-wrap">` or `<div class="kanye-west-wrapper">`

File: starter-code/index.html [hunk 8]
```diff
@@ -3,40 +3,81 @@
...
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
+    <nav>
```
Comment (ianizaguirre) [inline starter-code/index.html]: You are not writing any semantic tags in your HTML code? You are forgetting to wrap your nav with header tags. The `<header>` element is used to contain the sites name and the main navigation. 
You are also forgetting to include `<section>` tags. The `<section>` element groups related content together, typically each section would have its own heading.

**Comment by ianizaguirre** [comment]
I know you said in your comment, that you are still working on this. So I just left comments on what I have seen so far. I will leave this pull request OPEN since you are still working on it. When you are done working on it, Slack me a message so that I can check it for you.

Try to apply this feedback on this current project and your next project.
If you have any questions on my feedback, just send me a message on Slack. 
Aside from that, good job and keep up the good work! 😁

Kind Regards,
Ian Izaguirre | Ironhack TA

**Comment by rguasp** [comment]
i tried my best to improve on the mistakes you pointed out. For some reason i was having trouble using flexbox for the text and image alignment I know im not supposed to use margin but was out of ideas.
